solr/src/test/java/akka/stream/alpakka/solr/SolrTest.java

```diff
@@ -98,9 +103,12 @@ public void solrInputDocumentStream() throws Exception {
                 tuple -> {
                   Book book = tupleToBook.apply(tuple);
                   SolrInputDocument doc = bookToDoc.apply(book);
-                  return IncomingMessage.create(doc);
+                  List<IncomingMessage<SolrInputDocument, NotUsed>> list = new ArrayList<>();
+                  list.add(IncomingUpdateMessage.create(doc));
+                  return list;
```

Comment:
Instead of building the lists in user code, the examples should show `groupedWithin`.

Reply:
Yes it should